laws/src/main/scala/cats/laws/MonoidalLaws.scala

```diff
@@ -0,0 +1,16 @@
+package cats.laws
+
+import cats.Monoidal
+
+trait MonoidalLaws[F[_]] {
```

Comment:
Can we define the left/right identity and associativity laws here, but instead of returning `IsEq[X]`, return a tuple, because the first and second elements will have different but isomorphic types?  For example, the left identity law would return a `(F[(Unit, A)], F[A])`. This stresses that this type class is not lawless, even if we need some type of functor in order to do the equality check for most `F`s.